guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+
+There is other contextual information about payments, like the names of the transacting parties or the purpose, that is also useful to keep and communicate. Remember, the blockchain does not store any of this. Such information can only be stored in your application or encoded in [payment links](https://bitcoin.design/guide/foundations/wallet-interoperability/#payment-links). After a transaction is finalized, all that will get recorded are amounts and addresses.
+
+Let’s now explore our options when receivers may have other payment details that they may want to share with the sender or store alongside the payment request for their own record keeping. We will also look at the friction points that can arise from address incompatibility.
+
+<!-- 
+Update glossary
+
+Each address that a Bitcoin application generates has an accompaniying private key. This private key allows the receiver to unlock the bitcoin that is sent to the address when the receiver chooses to spend it.
+
+Although addresses are not a problem to share publically, their accompanying private keys which typically generated from a single recovery phrase must be kept secure and private.
+-->
+
+## Creating an invoice
+Oftentimes, to complete a payment, the sender may need to know the specific amount to be sent. On the receiver’s side, they may want to keep some notes about who sent the bitcoin to help them manage their funds in the future and understand their past transaction activity.
+
+The following are the primary properties of a Bitcoin invoice:
+
+- **Address** – The destination of the payment
+- **Amount** – How much is being requested (optional)
+- **From** – Name of who is sending the payment (optional)
+- **Purpose/Message** – Note with a description of what the payment is for (optional)
+
+While the application will generate a new address in the background, other information like; how much is being requested, the name of the sender, and what is being paid for has to be manually entered by the receiver. The order in which these properties get entered is up to you, and besides the address, all else is optional.
+
+<!--
+Technical nitpick;
+Addresses are not exactly "generated"
+since they are deterministic technically
+what happens is an unused address just gets chosen
+-->
+
+The more time passes, the more difficult it will be to recall the details of transactions. When transacting in Bitcoin there is no requirement for senders and receivers even to identify themselves, so both sides may end up having a transaction history with only amounts and random-looking text (addresses) to reference when trying to understand where some funds came from.
```

Comment:
```suggestion
The more time passes, the more difficult it will be to recall the details of transactions. When transacting in Bitcoin, there is no requirement for senders and receivers even to identify themselves, so both sides may end up having a transaction history with only amounts and random-looking text (addresses) to reference when trying to understand where some funds came from.
```